lib/chef/provider/mount/mount.rb

```diff
@@ -267,6 +246,35 @@ def mount_options_unchanged?
             @current_resource.pass == @new_resource.pass
         end
 
+        # It will update or delete the entry from fstab.
+        def edit_fstab(action = :update)
```

Comment:
Let's remove the default action value so it's more obvious above what a plain call to edit_fstab does. Ideally you can see where a method is called and understand what it's doing.

Reply:
@tas50 I have updated it, please have a look.